### PR TITLE
Xsoar engineer update - added body argument to xet-send-mail command.

### PR DESCRIPTION
**Pull request by beauchompers**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Added the body argument to the xet-send-mail command.   Command does nothing, just says it didn't send a fake email, used as part of our XSOAR Engineer Training videos from customer success.  

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by sapirshuker** [comment]
Hi @beauchompers,
Thank you for your contribution. 
Your PR review will be slightly delayed, I was sick, I apologize in advance. 
I'm available for any of your questions here or on slack.
Thanks again.

File: Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/XSOAREngineerTraining.yml [hunk 1]
```diff
@@ -97,6 +97,9 @@ script:
     - description: Who to send the fake email to.
       name: to
       required: true
+    - description: The body of the fake email that we don't be sending.
+      name: body
```
Comment (sapirshuker) [inline Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/XSOAREngineerTraining.yml:101]: Please generate a updated README.md file for your integration/script.
To do that, please run the `demisto-sdk generate-docs` command (use the -e flag for using a commad_examples file) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.

File: Packs/XSOAR_Engineer_Training/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### XSOAR Engineer Training
+
+- Added the *body* argument to the *xet-send-mail*
```
Comment (sapirshuker) [inline Packs/XSOAR_Engineer_Training/ReleaseNotes/1_0_1.md:6]: Do you mean to add the body argument without any changes to the .py file?
I see in the code that the given arguments have no meaning (The xet-send-mail command only prints a hardcoded string). Is that your intention?

**Review by sapirshuker — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Good work :)
Please see my comments.

File: Packs/XSOAR_Engineer_Training/ReleaseNotes/1_0_1.md (hunk 2 above)
Comment (beauchompers) [reply Packs/XSOAR_Engineer_Training/ReleaseNotes/1_0_1.md]: Yeah, the command doesn't really do anything, we use it for training customers as it simulates sending an email, without actually sending an email.  Updated the release notes file.

File: Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/XSOAREngineerTraining.yml (hunk 1 above)
Comment (beauchompers) [reply Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/XSOAREngineerTraining.yml:101]: Done.

File: Packs/XSOAR_Engineer_Training/ReleaseNotes/1_0_1.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (beauchompers) [reply Packs/XSOAR_Engineer_Training/ReleaseNotes/1_0_1.md]: Done.

**Comment by beauchompers** [comment]
> Thank you for your contribution! Good work :) Please see my comments.

Should all be resolved now, let me know if there is anything else needed.

For what it's worth the demisto-sdk failed to generate the command examples when using an XSOAR 8 instance, says they all failed, when I did the same command from an XSOAR 6.12 instance it worked.

**Review by sapirshuker — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Good work :)
Please see my comments.

File: Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/README.md [hunk 3]
```diff
@@ -38,6 +39,15 @@ Fetches events from the XSOAR Engineer Training (XET) integration.
 #### Context Output
 
 There is no context output for this command.
+#### Command example
+```!xet-get-events```
+#### Human Readable Output
+
+>### Training Events
+>|eventID|occurred|sourceIP|sourceUser|type|url|urlCategory|userAgent|
```
[the remark this answers is not in the document]
Comment (beauchompers) [reply Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/README.md:47]: There are no secrets here, again the integration is used for training purposes so all those usernames and information is made up (or in this case based on James Bond novels)

File: Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/command_examples.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+!xet-ad-get-user email="james.bond@xsoar.local"
+!xet-ad-set-new-password username="XSOAR007" password="bondjamesbond"
+!xet-ad-expire-password username="XSOAR007"
```
[the remark this answers is not in the document]
Comment (beauchompers) [reply Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/command_examples.txt:3]: Nope, all this is made up as it's used for training purposes.

File: Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/README.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 The XSOAR Engineer Training (XET) integration provides sample data to fetch events into Cortex XSOAR, and commands to build playbooks around.
 
-Use for training purposes only. 
+Use for training purposes only.
+This integration was integrated and tested with version xx of XSOAR Engineer Training.
```
[the remark this answers is not in the document]
Comment (beauchompers) [reply Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/README.md:4]: DOne.

**Review by sapirshuker — APPROVED** [review]
Great job!

**Pull request merged**